src/openms/source/ANALYSIS/XLMS/OpenPepXLLFAlgorithm.cpp

```diff
@@ -350,6 +356,20 @@ using namespace OpenMS;
     specGenParams_mainscore.setValue("add_k_linked_ions", "true");
     specGen_mainscore.setParameters(specGenParams_mainscore);
 
+    // use a less stringent tolerance for the tagger
+    double tagger_tol;
+    if (fragment_mass_tolerance_unit_ppm_) // ppm: increase tolerance by 50%
+    {
+      tagger_tol = fragment_mass_tolerance_xlinks_ + (fragment_mass_tolerance_xlinks_ / 2.0);
+    }
+    else // dalton: turn into ppm at 500 mz
+    {
+      tagger_tol = fragment_mass_tolerance_xlinks_ / 1e-6 / 500;
```

Comment:
dividing by 1e-6 is somewhat confusing.
How about using `Math::getPPM(500, 500+fragment_mass_tolerance_xlinks_)` ?